db/seeds.rb

```diff
@@ -1,12 +1,7 @@
-require 'seed_helper'
-
-include SeedHelper
-
-User.delete_all
-Practice.delete_all
-
 import_fixture 'companies'
 import_fixture 'users'
 import_fixture 'practices'
 import_fixture 'categories'
 import_fixture 'learnings'
+import_fixture 'questions'
+import_fixture 'answers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.